src/Compute/v2/Models/Limit.php

```diff
@@ -0,0 +1,21 @@
+<?php declare (strict_types = 1);
+
+namespace OpenStack\Compute\v2\Models;
+
+use OpenCloud\Common\Resource\OperatorResource;
+
+/**
+ * Represents a Compute v2 Limit
+ *
+ * @property \OpenStack\Compute\v2\Api $api
+ */
+class Limit extends OperatorResource
+{
+    /** @var string */
+    public $rate;
```

Comment:
are you sure this is a string? [the docs](http://developer.openstack.org/api-ref-compute-v2.1.html#showlimits) has it as an array